<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuranReciter</class>
 <widget class="QWidget" name="QuranReciter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>454</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Quran Reciter</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="lblVerseRange">
    <property name="text">
     <string>Verses:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnVerseFrom">
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>286</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblVerseRangeSeparator">
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnVerseTo">
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>286</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblCboChapter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chapter:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboChapter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboReciter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reciter:</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QCheckBox" name="chkRepeat">
    <property name="text">
     <string>Repeat</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnRepeat">
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="text">
     <string>times</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="lblVerse">
    <property name="text">
     <string>Verse:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnVerse"/>
  </widget>
  <widget class="QPushButton" name="btnReloadReciters">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Reload list of reciters</string>
   </property>
   <property name="text">
    <string>↻</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btnReplayCurrentVerse">
    <property name="toolTip">
     <string>Restart current verse</string>
    </property>
    <property name="text">
     <string>↶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="toolTip">
     <string>Stop</string>
    </property>
    <property name="text">
     <string>■</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlayPause">
    <property name="toolTip">
     <string>Play / Pause</string>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QSlider" name="volSlider">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
